<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIDebugCPU</class>
 <widget class="QDialog" name="UIDebugCPU">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debug CPU</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_13">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="gbRegisters">
       <property name="title">
        <string>Registers</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QListWidget" name="lwRegisters">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Courier New</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QGroupBox" name="gbDisassembledCode">
         <property name="title">
          <string>Disassembled Code</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="UIDisasm" name="lwDisassembledCode" native="true">
            <property name="minimumSize">
             <size>
              <width>416</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier New</family>
              <pointsize>9</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbBackTrace">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Other Debug</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QTabWidget" name="tabOtherDebug">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="tabPosition">
             <enum>QTabWidget::South</enum>
            </property>
            <property name="tabShape">
             <enum>QTabWidget::Rounded</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="tabsClosable">
             <bool>false</bool>
            </property>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>Backtrace</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_14">
              <item>
               <widget class="QListWidget" name="lwBackTrace">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Courier New</family>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_4">
             <attribute name="title">
              <string>Track Inf Loop Results</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_15">
              <item>
               <widget class="QTableWidget" name="twTrackInfLoop">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="showDropIndicator" stdset="0">
                 <bool>false</bool>
                </property>
                <property name="dragDropOverwriteMode">
                 <bool>false</bool>
                </property>
                <column>
                 <property name="text">
                  <string>Address</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Count</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QGroupBox" name="gbCodeBreakpoints">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Code Breakpoints</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLineEdit" name="leCodeBreakpoint">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="inputMask">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListWidget" name="lwCodeBreakpoints">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPushButton" name="pbAddCodeBreakpoint">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbDelCodeBreakpoint">
                <property name="text">
                 <string>Del</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbMemoryBreakpoints">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Memory Breakpoints</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLineEdit" name="leMemoryBreakpoint">
                  <property name="inputMask">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListWidget" name="lwMemoryBreakpoints">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPushButton" name="pbAddMemoryBreakpoint">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbDelMemoryBreakpoint">
                <property name="text">
                 <string>Del</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QCheckBox" name="cbRead">
                <property name="text">
                 <string>Read</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QCheckBox" name="cbWrite">
                <property name="text">
                 <string>Write</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QCheckBox" name="cbReadByte">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Byte</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbReadWord">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Word</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbReadLong">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Long</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbWriteByte">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Byte</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbWriteWord">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Word</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbWriteLong">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Long</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="pbStepInto">
       <property name="text">
        <string>Step Into</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbStepOver">
       <property name="text">
        <string>Step Over</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbStepOut">
       <property name="text">
        <string>Step Out</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbMemoryTransfer">
       <property name="text">
        <string>Memory Transfer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbMemoryEditor">
       <property name="text">
        <string>Memory Editor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbReserved1">
       <property name="text">
        <string>Reserved1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbReserved2">
       <property name="text">
        <string>Reserved2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbReserved3">
       <property name="text">
        <string>Reserved3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbReserved4">
       <property name="text">
        <string>Reserved4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbReserved5">
       <property name="text">
        <string>Reserved5</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dbbButtons">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UIDisasm</class>
   <extends>QWidget</extends>
   <header>UIDisasm.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>dbbButtons</sender>
   <signal>rejected()</signal>
   <receiver>UIDebugCPU</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
